internal/client/client.go

```diff
@@ -57,9 +57,9 @@ func GetVersion() string {
 	return cliVersion
 }
 
-// NewAuthClient function is returning a new AuthApiClient Client
+// NewAuthApiClient NewAuthClient function is returning a new AuthApiClient Client
 func NewAuthApiClient() (*AuthApiClient, error) {
-	url, err := ParseURL(viper.GetString("host"))
+	parseURL, err := ParseURL(viper.GetString("host"))
```

Comment:
This can simply be url right?

Reply:
<img width="500" alt="Screenshot 2024-11-25 at 6 23 59 PM" src="https://github.com/user-attachments/assets/15e02222-7aa8-46df-92c5-22ae7e00a57a">
my IDE warns me to rename it.